src/toolkits/activity_classification/activity_classifier.cpp

```diff
@@ -636,12 +639,19 @@ std::unique_ptr<data_iterator> activity_classifier::create_iterator(
   data_params.prediction_window = read_state<flex_int>("prediction_window");
   data_params.predictions_in_chunk = NUM_PREDICTIONS_PER_CHUNK;
   data_params.use_data_augmentation = use_data_augmentation;
-  data_params.random_seed = read_state<int>("random_seed");
+  if (!use_data_augmentation) {
```

Comment:
Nit: it would be more readable to switch the if and else cases and have the condition just be `if (use_data_augmentation)`